### DISCUSSION TITLE
Authorizing multiple permission from controller

### DISCUSSION THREAD

**Discussion by Enigmatec** [spatie/laravel-permission#2071] [Q&A]
How can i pass multiple permission to the can or cannot method in my controller.

this works for a single permission
  public function allUser(Request $request) {
        abort_if(auth()->user()->can('edit'), 403);
        return User::all();
    }
How can i pass multiple permission to the can  method...
Thanks

**Comment by erikn69** [comment]
```php
app(\Illuminate\Contracts\Auth\Access\Gate::class)->any(['view','edit'])
```
